src/app/static/filtered-question-list/filtered-question-list.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnChanges, SimpleChanges } from '@angular/core';
```

Comment:
As discussed, I think let's rework this setup to let preview-questions uncollapse themselves into full view-question components.  It seems a little easier and cleaner than passing information about what questions should be displayed/removed from various components on this page.  Feel free to bring up concerns here, ofc